n2y/plugins/dbfootnotes.py

```diff
@@ -0,0 +1,50 @@
+import logging
+
+from pandoc.types import Note
+
+from n2y.database import Database
+from n2y.errors import UseNextClass
+from n2y.mentions import PageMention
+
+plugin_data_key = "n2y.plugins.dbfootnotes"
+
+logger = logging.getLogger(__name__)
+
+
+class PageMentionFootnote(PageMention):
+    def __init__(self, client, notion_data, plain_text, block=None):
```

Comment:
I'd probably raise `UseNextClass` if `block` is None... this may happen implicitly already since perhaps `get_page` will fail...

Reply:
:+1: Fixed.